dockerfiles/keycloak/Dockerfile

```diff
@@ -10,7 +10,14 @@ FROM jboss/keycloak:6.0.1
 ADD che /opt/jboss/keycloak/themes/che
 ADD . /scripts/
 ADD cli /scripts/cli
+RUN ln -s /opt/jboss/tools/docker-entrypoint.sh
+RUN curl --location https://github.com/davidfestal/KEYCLOAK-10169-OpenShift4-User-Provider/releases/download/6.0.1-openshift-v4-provider/openshift4-extension-6.0.1.jar -o /opt/jboss/keycloak/standalone/deployments/openshift4-extension-6.0.1.jar
```

Comment:
we don't have jar on something that is not belonging to a given user (davidfestal ? )

Reply:
maybe use a multi-staged build ?

Reply:
eventually the fix will be in Keycloak upstream (and RH-SSO downstream) but for now it's a custom jar

Reply:
+1 to @benoitf suggestion. And consider using `curl -sSL` flags to follow redirections and avoid having the % for the download in the logs.